ansible-collection/plugins/modules/px_backup/backup.py

```diff
@@ -784,6 +759,13 @@ def inspect_backup(module: AnsibleModule, client: PXBackupClient) -> Dict[str, A
 
 def delete_backup(module: AnsibleModule, client: PXBackupClient) -> Tuple[Dict[str, Any], bool]:
     """Delete a backup"""
+    result = {  # Initialize a default result to avoid uninitialized references
```

Comment:
need to understand the use case which caused delete to not fall in the exception. Let's access the situation once again and see if this can be removed.

Reply:
Removed the api changes for now